scripts/README.md

```diff
@@ -170,8 +168,6 @@ This is a helper script for Knative E2E test scripts. To use it:
    cluster. You can also use the following boolean (0 is false, 1 is true)
```

Comment:
Remove the last sentence of this paragraph.

Reply:
Done.